src/outputFunctions.f90

```diff
@@ -294,7 +294,7 @@ subroutine outputRates( rSpecies, r, arrayLen, t, p, flag )
           ! r contains the occurences of each of the detailed species in reactions.
           ! r should have row lengths as in arrayLen, so all accesss to r(i,j) should
           ! be valid.
-          write (output_file_number, '(ES15.6E3, I14, A12, I15, ES15.6E3, A, A)') t, rSpecies(i), &
+          write (output_file_number, '(ES15.6E3, I14, A50, I15, ES15.6E3, A, A)') t, rSpecies(i), &
```

Comment:
Could you please amend to 52, in order to ensure there's always whitespace between columns, even when the full 50 characters are used.